tests/PhpcsScanScanCommitTest.php

```diff
@@ -896,4 +924,98 @@ public function testDoScanTest5() {
 			$issues_stats
 		);
 	}
+
+	public function testLintWontScanNumberOfLinesInvalid() {
+		$options_test = vipgoci_unittests_options_test(
+			$this->options,
+			array( 'phpcs-runtime-set', 'github-token', 'token' ),
+			$this
+		);
+
+		if ( -1 === $options_test ) {
+			return;
+		}
+
+		$this->options['commit'] =
+			$this->options['commit-test-phpcs-scan-commit-5'];
+
+		$this->options['phpcs-skip-scanning-via-labels-allowed'] =
+			false;
+
+		$issues_submit = array();
+		$issues_stats = array();
+		$issues_skipped = array();
+
+		vipgoci_unittests_output_suppress();
+
+		$prs_implicated = vipgoci_github_prs_implicated(
+			$this->options['repo-owner'],
+			$this->options['repo-name'],
+			$this->options['commit'],
+			$this->options['github-token'],
+			$this->options['branches-ignore']
+		);
+
+
+		foreach( $prs_implicated as $pr_item ) {
+			$issues_stats[
+			$pr_item->number
+			][
+			'error'
+			] = 0;
+
+			$issues_skipped[ $pr_item->number ] = $this->getDefaultSkippedFilesDueIssuesMock();
+		}
+
+		$issues_skipped[39] = array(
+			'issues' => array(
+				'max-lines' => array ( 'test1/myfile-1.php' )
+			),
+			'total' => 1
+		);
+
+		$this->options['local-git-repo'] =
+			vipgoci_unittests_setup_git_repo(
+				$this->options
+			);
+
+		if ( false === $this->options['local-git-repo'] ) {
+			$this->markTestSkipped(
+				'Could not set up git repository: ' .
+				vipgoci_unittests_output_get()
+			);
+
+			return;
+		}
+
+		vipgoci_phpcs_scan_commit(
+			$this->options,
+			$issues_submit,
+			$issues_stats,
+			$issues_skipped
+		);
+
+		vipgoci_unittests_output_unsuppress();
+
+		$expected_issues_skipped = array(
+			39 => array(
+				'issues' => array(
+					'max-lines' => array ( 'test1/myfile-1.php' )
+				),
+				'total' => 1
+			)
+		);
+
```

Comment:
Suggest to a) add a new file to the commit that would be scanned and has issues, b) to add some code with problems to the file that is skipped, and c) and then check here if `$issues_submit` contains only issues for the first file. This way we know that PHPCS scanning returns with issues even if some files were skipped, and we know that that they are in the correct file.

This would make the test a bit more comprehensive.